Sources/App/configure.swift

```diff
@@ -14,8 +14,9 @@ private let jiraProjects = [
     "COREUS": 17127, // Babylon US Core Product
     "COVIDBUGS": 17459, // COVID-19 hotfix bugs
     "CW"    : 16832, // Consumer Web
-    "ETA"   : 17369, // Engagement Tribe - Activate
-    "ETR"   : 17372, // Engagement Tribe - Retain
+    "ETA"   : 17369, // Engagement Tribe - Activate - TO BE REMOVED ONCE MIGRATED TO EXPCORE
+    "ETR"   : 17372, // Engagement Tribe - Retain - TO BE REMOVED AFTER 4.21.0 CUT
+    "EXPCORE"   : 17372, // Experience Tribe - Core Apps
```

Comment:
Wait no, how come EXPCORE, new JIRA board, has the same ID as old ETR board?

Reply:
https://github.com/babylonhealth/Stevenson/pull/56#issuecomment-616568390